src/kibana/components/vislib/visualizations/_point_series_chart.js

```diff
@@ -100,22 +102,19 @@ define(function (require) {
 
       function callPlay(event) {
         var boundData = event.target.__data__;
+        var offsetX = $chartEl.offset().left;
         var wholeBucket = boundData && boundData.x != null;
 
+        // the min and max that the endzones start in
         var min = leftEndzone.w;
         var max = rightEndzone.x;
 
         // bounds of the cursor to consider
-        var xLeft = event.offsetX;
-        var xRight = event.offsetX;
-        if (wholeBucket) {
-          xLeft = xScale(boundData.x);
```

Comment:
Idk man it does that now. It seemed to me this is what the function was
really trying to accomplish
On May 26, 2015 4:43 PM, "Spencer" <notifications@github.com> wrote:

> In src/kibana/components/vislib/visualizations/_point_series_chart.js
> <https://github.com/elastic/kibana/pull/3980#discussion_r31077611>:
>
> >          var min = leftEndzone.w;
> >          var max = rightEndzone.x;
> >
> >          // bounds of the cursor to consider
> > -        var xLeft = event.offsetX;
> > -        var xRight = event.offsetX;
> > -        if (wholeBucket) {
> > -          xLeft = xScale(boundData.x);
>
> That code is what's responsible for showing the warning anytime you hover
> on a bucket that goes out of bounds.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/kibana/pull/3980/files#r31077611>.
>